src/ViewController.js

```diff
@@ -200,7 +200,19 @@ class ViewController extends HasModels {
     })
   }
 
-  _modelAttributes(withRelated=false) {
+  /*
+  * Override to modify the related objects find for related attributes
+  * Use this to filter by user
+  * @param {String} modelName 
+  * @param {Object} find default find() for the related model (returned unmodified by default)
+  * @param {Request} req The express req object
+  * @returns {Object) the find() to use
+  */
+  relatedInstances(modelName, find, req) {
+    return find
+  }
+
+  _modelAttributes(withRelated=false, req=null) {
```

Comment:
Req is optional (there are various other uses of this that are not interested in related at all), probably a better refactor would split out the related part of this entirely to its own method.